### ISSUE TITLE
Login issue in macOS 10.11

### ISSUE THREAD

**Issue by Alex-C-Eltima** [pCloud/pcloud-sdk-swift#6]
Hello, 

We are using your SDK to connect to pCloud from [CloudMounter](https://cloudmounter.net) app, and have the following behavior, when trying to connect under macOS 10.11:

![image](https://user-images.githubusercontent.com/17063982/85711656-2822d880-b6f0-11ea-8edf-f1d5f1b30ea6.png)

Any suggestion will be appreciated.

**Comment by tmpit** [comment]
Hey, from the screenshot it looks like the web page doesn't fully load. I just tried going through authorization in a sample macOS app and the web page loaded fine. Can you tell me which version of the SDK you are using and if this only happens under macOS 10.11? Also, since when were you observing this behaviour?

**Comment by Alex-C-Eltima** [comment]
Hello, thank you for reply. 

We are using latest release version of your SDK 3.0.0, but it also occurs on the earlier versions, and happens only under macOS 10.11. 

We can't exactly specify since when this behavior occurs, as it was working correctly when we added support of pCloud to our app. Last week one of our users reported this problem in outdated version of our app, that used earlier version of your SDK. And after that we confirmed it on the latest version as well.
